docs/README.md

```diff
@@ -51,3 +51,24 @@ Removes an item from the cart. `props` must contain either the `uniqueId` or `in
 ```tsx
 removeItem({ uniqueId: 'E1FDB9F661D74543AE3A13D587641E63' })
 ```
+
+### `addItemToCart: (props: [Item]) => Promise<{data: OrderForm, error: boolean}>`
```

Comment:
Does this need to have a return value? I'd rather keep consistency with the other API methods, which return `void`. Also, `error` being a `boolean` is weird, it gives no context about what went wrong.

Reply:
I'm also unsure about `error` being a `boolean`, but I also think that the orchestrators should  deal with a potential error and just inform to the UI component that something happened. But as I said I'm not sure also. I talked to @klzns about that but we couldn't come to a conclusion.

Reply:
When it comes to the method returning `void`, I have a question: Some UI components like `add-to-cart-button` needs to know the updated `orderForm` to compare with the last one and, with that, show contextualized toast messages like "An Item was added to the cart", "The Item is already on the cart". How could this be done instead of making `addItemToCart` to return the new `orderForm`?

Reply:
Hm... It can retrieve the new `orderForm` from `OrderManager`, wouldn't that work?

Reply:
How could it know if the `orderForm` he's retrieving from `OrderManager` is the one resultant from the mutation? The component should have to know if the mutation worked properly and, with that, it could compare the two `orderForm`'s to show the contextualized messages.

I don't see how to accomplish that with `addItemToCart` returning `void`.

Reply:
I think one solution that adheres to our paradigm is moving this toast logic to our GraphQL, adding the toast message to the `messages` field of our orderForm and displaying them in the front end.
I don't want to have our APIs returning data because it kinda goes against the idea of making the interface components "dumb", but if it ends up being necessary then we should return data in all the other methods as well to maintain consistency.